.stylelintrc.json

```diff
@@ -1,5 +1,8 @@
 {
 	"extends": "stylelint-config-wordpress",
+	"ignoreFiles": [
+        "./css/main.scss"
```

Comment:
This didn't work for me but `"./vendor/**/*.scss"` did.

Reply:
weird that is what I had, but must have mixed up my commit, thanks Ron I'll fix that up